### PR TITLE
Updating publications list

### PR DESCRIPTION
**Pull request by cipollone**

### CODE REVIEW

File: publications.bib [hunk 1]
```diff
@@ -17,7 +17,382 @@ @inproceedings{citing-name
...
+	volume={37},
+	DOI={10.1609/aaai.v37i13.27068},
+	number={13},
+	journal={Proceedings of the AAAI Conference on Artificial Intelligence (AAAI)},
+	year={2023},
+	pages={16428-16430},
+  note={System Demonstration.},
+	url_paper={https://ojs.aaai.org/index.php/AAAI/article/view/27068},
+}
+
```
Comment (francescofuggitti) [inline publications.bib:263]:
```suggestion

@article{icaps2023fc,
	author={Fuggitti, Francesco and Chakraborti, Tathagata},
	title={NL2LTL – a Python Package for Converting Natural Language (NL) Instructions to Linear Temporal Logic (LTL) Formulas},
	journal={{ICAPS}},
	year={2023},
        note={System Demonstration.},
        bibbase_note = {<span style="color: red">Best System Demonstration Runner-Up.</span>} 
}
```

File: publications.bib [hunk 2]
```diff
@@ -17,7 +17,382 @@ @inproceedings{citing-name
...
+@inproceedings{aaai2023adfmmp,
+  author       = {Giacomo Acitelli and Giuseppe {De Giacomo} and Francesco Fuggitti and {Fabrizio Maria} Maggi and Andrea Marrella and Fabio Patrizi},
+  title        = {Declarative Trace Alignment via Automated Planning (Extended Abstract)},
+  booktitle    = {{AI4BPM} at {AAAI}},
+  year         = {2023},
+  url_paper    = {https://whitemech.github.io/papers/2023/aaai2023adfmmp.pdf},
+}
+
+@inproceedings{icaps2023bdffgs,
+	author={Bonassi, Luigi and De Giacomo, Giuseppe and Favorito, Marco and Fuggitti, Francesco and Gerevini, Alfonso Emilio and Scala, Enrico},
```
Comment (francescofuggitti) [inline publications.bib:273]:
```suggestion
	author={Bonassi, Luigi and {De Giacomo}, Giuseppe and Favorito, Marco and Fuggitti, Francesco and Gerevini, Alfonso Emilio and Scala, Enrico},
```

File: publications.bib [hunk 3]
```diff
@@ -17,7 +17,382 @@ @inproceedings{citing-name
...
+  title={Planning for Temporally Extended Goals in Pure-Past Linear Temporal Logic},
+	volume={33},
+	DOI={10.1609/icaps.v33i1.27179},
+	number={1},
+	journal={Proceedings of the International Conference on Automated Planning and Scheduling},
```
Comment (francescofuggitti) [inline publications.bib:278]:
```suggestion
	journal={Proceedings of the International Conference on Automated Planning and Scheduling ({ICAPS})},
```

File: publications.bib [hunk 4]
```diff
@@ -17,7 +17,382 @@ @inproceedings{citing-name
...
+  booktitle={{RPA} forum at {BPM}},
+  year={2022},
+  publisher={Springer International Publishing},
+  pages={123--137},
+  doi={https://doi.org/10.1007/978-3-031-16168-1_8},
+  url_paper={https://link.springer.com/chapter/10.1007/978-3-031-16168-1_8}
+}
+
+@article{aaai2023rkd,
+	author={Ronca, Alessandro and Knorozova, Nadezda Alexandrovna and De Giacomo, Giuseppe},
```
Comment (francescofuggitti) [inline publications.bib:308]:
```suggestion
	author={Ronca, Alessandro and Knorozova, Nadezda Alexandrovna and {De Giacomo}, Giuseppe},
```

File: publications.bib [hunk 5]
```diff
@@ -17,7 +17,382 @@ @inproceedings{citing-name
...
+	title={Automata Cascades: Expressivity and Sample Complexity},
+	volume={37},
+	DOI={10.1609/aaai.v37i8.26147},
+	number={8},
+	journal={Proceedings of the AAAI Conference on Artificial Intelligence (AAAI)},
```
Comment (francescofuggitti) [inline publications.bib:313]:
```suggestion
	journal={Proceedings of the AAAI Conference on Artificial Intelligence ({AAAI})},
```

File: publications.bib [hunk 6]
```diff
@@ -17,7 +17,382 @@ @inproceedings{citing-name
...
+  number = {4},
+  article-number = {236},
+  url_paper = {https://www.mdpi.com/2504-446X/7/4/236},
+  issn = {2504-446X},
+  doi = {10.3390/drones7040236}
+}
+
+@inproceedings{caise2023dflmms,
+  author       = {De Giacomo, Giuseppe and Favorito, Marco and Leotta, Francesco and Mecella, Massimo and Monti, Flavia and Silo, Luciana},
+  title        = {AIDA: A Tool for Resiliency in Smart Manufacturing},
```
Comment (francescofuggitti) [inline publications.bib:344]:
```suggestion
  author       = {{De Giacomo}, Giuseppe and Favorito, Marco and Leotta, Francesco and Mecella, Massimo and Monti, Flavia and Silo, Luciana},
  title        = {{AIDA}: A Tool for Resiliency in Smart Manufacturing},
```

File: publications.bib [hunk 7]
```diff
@@ -17,7 +17,382 @@ @inproceedings{citing-name
...
+  booktitle    = {{IEEE} International Conference on Web Services, {ICWS} 2023},
+  publisher    = {{IEEE}},
+  year         = {2023},
+  url_paper    = {https://whitemech.github.io/papers/2023/icws2023mslm.pdf},
+  bibbase_note = {<span style="color: red">To appear.</span>},
+}
+
+@article{cii2023dflms,
+  title={Digital twins composition in smart manufacturing via Markov decision processes},
+  author={De Giacomo, Giuseppe and Favorito, Marco and Leotta, Francesco and Mecella, Massimo and Silo, Luciana},
```
Comment (francescofuggitti) [inline publications.bib:376]:
```suggestion
  author={{De Giacomo}, Giuseppe and Favorito, Marco and Leotta, Francesco and Mecella, Massimo and Silo, Luciana},
```

File: publications.bib [hunk 8]
```diff
@@ -92,38 +484,47 @@ @inproceedings{ijcai2022dflps
...
-  author = {Diego Calvanese and
-            Giuseppe {De Giacomo} and
-            Marco Montali and
-            Fabio Patrizi},
-  year = {2022},
-  booktitle = {Proceedings of the Thirty-First International Joint Conference on Artificial Intelligence ({IJCAI})},
-  url_paper = {https://whitemech.github.io/papers/2022/ijcai2022cdmp.pdf},
-  note = {To appear},
+  author       = {Diego Calvanese and
+                  Giuseppe De Giacomo and
```
Comment (francescofuggitti) [inline publications.bib:488]:
```suggestion
                  Giuseppe {De Giacomo} and
```

File: publications.bib [hunk 9]
```diff
@@ -92,38 +484,47 @@ @inproceedings{ijcai2022dflps
...
-  author = {Benjamin Aminof and
-            Giuseppe {De Giacomo} and
-            Sasha Rubin and
-            Florian Zulegerand},
-  year = {2022},
-  booktitle = {Proceedings of the Thirty-First International Joint Conference on Artificial Intelligence ({IJCAI})},
-  url_paper = {https://whitemech.github.io/papers/2022/ijcai2022adrz.pdf},
-  note = {To appear},
+  author       = {Benjamin Aminof and
+                  Giuseppe De Giacomo and
```
Comment (francescofuggitti) [inline publications.bib:503]:
```suggestion
                  Giuseppe {De Giacomo} and
```

File: publications.bib [hunk 10]
```diff
@@ -92,38 +484,47 @@ @inproceedings{ijcai2022dflps
...
-  author = {Natasha Alechina and
-            Giuseppe {De Giacomo} and
-            Brian Logan and
-            Giuseppe Perelli},
-  year = {2022},
-  booktitle = {Proceedings of the 19th International Conference on Principles of Knowledge Representation and Reasoning ({KR})},
-  url_paper = {https://whitemech.github.io/papers/2022/kr2022adlp.pdf},
-  note = {To appear},
+  author       = {Natasha Alechina and
+                  Giuseppe De Giacomo and
```
Comment (francescofuggitti) [inline publications.bib:518]:
```suggestion
                  Giuseppe {De Giacomo} and
```

File: publications.bib [hunk 11]
```diff
@@ -216,20 +615,26 @@ @inproceedings{gandalf2021ztpv
...
 }
 @inproceedings{ijcai2022zd,
-  title = {Synthesis of Maximally Permissive Strategies for LTLf Specifications},
-  author = {Shufang Zhu and Giuseppe {De Giacomo}},
-  year = {2022},
-  booktitle = {Proceedings of the Thirty-First International Joint Conference on Artificial Intelligence ({IJCAI})},
-  url_paper = {https://whitemech.github.io/papers/2022/ijcai2022zd.pdf},
-  note = {To appear},
+  author       = {Shufang Zhu and
+                  Giuseppe De Giacomo},
```
Comment (francescofuggitti) [inline publications.bib:619]:
```suggestion
                  Giuseppe {De Giacomo}},
```

File: publications.bib [hunk 12]
```diff
@@ -216,20 +615,26 @@ @inproceedings{gandalf2021ztpv
...
+  title        = {Synthesis of Maximally Permissive Strategies for LTLf Specifications},
```
Comment (francescofuggitti) [inline publications.bib:620]:
```suggestion
  title        = {Synthesis of Maximally Permissive Strategies for {LTLf} Specifications},
```

File: publications.bib [hunk 13]
```diff
@@ -216,20 +615,26 @@ @inproceedings{gandalf2021ztpv
...
+
 @inproceedings{kr2022zd,
-  title = {Act for Your Duties but Maintain Your Rights},
-  author = {Shufang Zhu and Giuseppe {De Giacomo}},
-  year = {2022},
-  booktitle = {Proceedings of the 19th International Conference on Principles of Knowledge Representation and Reasoning ({KR})},
-  url_paper = {https://whitemech.github.io/papers/2022/kr2022zd.pdf},
-  note = {To appear},
+  author       = {Shufang Zhu and
+                  Giuseppe De Giacomo},
```
Comment (francescofuggitti) [inline publications.bib:631]:
```suggestion
                  Giuseppe {De Giacomo}},
```

File: publications.bib [hunk 14]
```diff
@@ -261,21 +666,36 @@ @BPM
   url_paper = {https://whitemech.github.io/papers/2021/itbpm2021dflms.pdf},
 }
 @inproceedings{ijcai2022dflvxz,
-  title = {LTLf Synthesis as AND-OR Graph Search: Knowledge Compilation at Work},
-  author = {{De Giacomo}, Giuseppe and Favorito, Marco and Li, Jianwen and Vardi, Moshe and Xiao, Shengping and Zhu, Shufang},
-  year = {2022},
-  booktitle = {Proceedings of the Thirty-First International Joint Conference on Artificial Intelligence ({IJCAI})},
-  url_paper = {https://whitemech.github.io/papers/2022/ijcai2022dflvxz.pdf},
-  note = {To appear},
+  author       = {Giuseppe De Giacomo and
```
Comment (francescofuggitti) [inline publications.bib:669]:
```suggestion
  author       = {Giuseppe {De Giacomo} and
```

File: publications.bib [hunk 15]
```diff
@@ -261,21 +666,36 @@ @BPM
...
+                  Marco Favorito and
+                  Jianwen Li and
+                  Moshe Y. Vardi and
+                  Shengping Xiao and
+                  Shufang Zhu},
+  title        = {LTLf Synthesis as {AND-OR} Graph Search: Knowledge Compilation at Work},
```
Comment (francescofuggitti) [inline publications.bib:675]:
```suggestion
  title        = {{LTLf} Synthesis as {AND-OR} Graph Search: Knowledge Compilation at Work},
```

File: publications.bib [hunk 16]
```diff
@@ -261,21 +666,36 @@ @BPM
...
 @inproceedings{ijcai2022rpd,
-  author = {Alessandro Ronca and Gabriel {Paludo Licks} and Giuseppe {De Giacomo}},
-  title = {Markov Abstractions for {PAC} Reinforcement Learning in Non-Markov Decision Processes},
-  booktitle = {Proceedings of the Thirty-First International Joint Conference on Artificial Intelligence ({IJCAI})},
-  year = {2022},
-  url_paper = {https://whitemech.github.io/papers/2022/ijcai2022rpd.pdf},
-  note = {To appear},
+  author       = {Alessandro Ronca and
+                  Gabriel Paludo Licks and
+                  Giuseppe De Giacomo},
```
Comment (francescofuggitti) [inline publications.bib:688]:
```suggestion
                  Gabriel {Paludo Licks} and
                  Giuseppe {De Giacomo}},
```

File: publications.bib [hunk 17]
```diff
@@ -261,21 +666,36 @@ @BPM
...
+  title        = {Markov Abstractions for {PAC} Reinforcement Learning in Non-Markov
```
Comment (francescofuggitti) [inline publications.bib:689]:
```suggestion
  title        = {Markov Abstractions for {PAC} Reinforcement Learning in Non-Markovian
```

File: publications.bib [hunk 18]
```diff
@@ -294,13 +714,19 @@ @inproceedings{aaai22cmp
...
-  author = {Ramon {Fraga Pereira} and Andre {Grahl Pereira} and Frederico Messa and Giuseppe {De Giacomo}},
-  title = {Iterative {Depth-First} Search for Fully Observable Non-Deterministic Planning},
-  booktitle = {Proceedings of the Thirty-Second International Conference on Automated Planning and Scheduling ({ICAPS})},
-  year = {2022},
-  url_paper = {https://whitemech.github.io/papers/2022/icaps2022fgmd.pdf},
-  note = {To appear}
+  author       = {Ramon Fraga Pereira and
+                  Andr{\'{e}} Grahl Pereira and
+                  Frederico Messa and
+                  Giuseppe De Giacomo},
```
Comment (francescofuggitti) [inline publications.bib:720]:
```suggestion
  author       = {Ramon {Fraga Pereira} and
                  Andr{\'{e}} {Grahl Pereira} and
                  Frederico Messa and
                  Giuseppe {De Giacomo}},
```

**Review by francescofuggitti — CHANGES_REQUESTED** [review]
Thank you! I just made a couple of fixes

File: publications.bib (hunk 17 above)
Comment (cipollone) [reply publications.bib:717]: Non-Markov seems correct here

**Review by cipollone — COMMENTED** [review]
Applied suggestions (groupings).

**Review by francescofuggitti — APPROVED** [review]
That's great!

**Pull request merged**